adapter/redisQuota/redisQuota.go

```diff
@@ -0,0 +1,326 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package redisQuota provides a quota implementation with redis as backend.
+// The prerequisite is to have a redis server running.
+
+package redisQuota
```

Comment:
The package name should be all lowercase.

We haven't been as vigilant as we should have been early in mixer about this, but we should start fixing this now.

https://blog.golang.org/package-names
> Good package names are short and clear. They are lower case, with no under_scores or mixedCaps.